### PR TITLE
Survey Form

### PR DESCRIPTION
**Pull request by mutonic**
In this project we have added the following

- Add the HTML Skeleton of a survey form
- Add the styling of the form

### CODE REVIEW

**Comment by mutonic** [comment]
STATUS: REQUEST CHANGES ♻️

Hey,

It was a good job, but there are some issues that could make this project better, but you are almost there



#Higlight

- The correct flow were used  ✅
- Have added the MIT licence  ✅
- Descriptive PR summary ✅ 
- Descriptive PR title ✅ 


## Required Changes ♻️
- In my opinion, it would greatly improve the quality of the code to incorporate **linters** that can detect and flag errors made in HTML and CSS.
-  Remember to use **meaningful commit messages** so that in the future, you will be able to remember what changes were made.
- Remember to remove unnecessary spaces in the code to improve its readability.
- Make sure to include a README file that describes what you have done in the project.

## Optional suggestions
-For future projects, I would suggest **not working on the main branch.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @mutonic 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Good use of summary template ✔️ 

## Required Changes ♻️

**- [ ] Please kindly ensure to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and make sure you leave at least two comments under a line or block of code that fits the context the comment is about.**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @mutonic 👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅.

- [x] Nice work suggesting some changes👍.
- [x] Review Summary looks great👍.


 ## Required Changes ♻️

- [ ] Just as[ the previous reviewer has suggested](https://github.com/mutonic/trials-review/pull/1#pullrequestreview-1407498816), we are required to make use of github inline comments feature to point out some of the changes you are suggesting👍 Kindly take a look at [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) and warch [this video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) to learn how to make use of the github inline comment feature and make the necessary change👍

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        </div>
+        
+        <div class="content">
+            <label id="email-label" for="email">Email</label>
+            <input type="email" id="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your email" required>
+            
+
+        </div>
+       
+        <div class="content">
```
Comment (mutonic) [inline index.html:33]: - [ ]  Remove unnecessary spaces to make the code more readable.
- [ ] I would suggest that you use fewer "id" selectors and more "class" selectors.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg)
+    
+      
+    }
+
```
Comment (mutonic) [inline style.css:16]: - [ ] As in html Remove unnecessary spaces to make the code more readable.
- [ ] Remember to use indentation to make your code clearer.

**Review by mutonic — COMMENTED** [review]
STATUS: REQUEST CHANGES ♻️

Hey,

It was a good job, but there are some issues that could make this project better, but you are almost there



#Higlight

- The correct flow were used  ✅
- Have added the MIT licence  ✅
- Descriptive PR summary ✅ 
- Descriptive PR title ✅ 


## Required Changes ♻️
- In my opinion, it would greatly improve the quality of the code to incorporate **linters** that can detect and flag errors made in HTML and CSS.
-  Remember to use **meaningful commit messages** so that in the future, you will be able to remember what changes were made.
- Remember to remove unnecessary spaces in the code to improve its readability.
- Make sure to include a README file that describes what you have done in the project.

## Optional suggestions
-For future projects, I would suggest **not working on the main branch.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @mutonic, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**